<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>itemWidget</class>
 <widget class="QWidget" name="itemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>60</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>36</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1065</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>file item</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="IconWidget" name="niiStateIcon">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>5</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>5</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="BodyLabel" name="nameLabel">
        <property name="text">
         <string>name</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="IndeterminateProgressRing" name="loadingRing">
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="strokeWidth">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ProgressRing" name="loadedRing">
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="strokeWidth">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="HorizontalSeparator" name="HorizontalSeparator"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IconWidget</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>HorizontalSeparator</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ProgressBar</class>
   <extends>QProgressBar</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ProgressRing</class>
   <extends>ProgressBar</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>IndeterminateProgressRing</class>
   <extends>QProgressBar</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
